app/views/admin/donations/index.slim

```diff
@@ -0,0 +1,20 @@
+- title t('donations.plural')
+
+table.ui.very.basic.table
+  thead
+    tr
+      th.center.aligned = t 'donations.donator'
+      th.center.aligned = t 'goals.amount'
+      th.center.aligned = t 'words.date'
+
+  tbody
+    - donations.each do |donation|
+      tr
+        - if donation.user
+          td.center.aligned = "#{donation.user.first_name} #{donation.user.last_name}"
+        - else
+          td.center.aligned = "-"
+        td.center.aligned = donation.amount.to_f
+        td.center.aligned = donation.created_at.strftime("%d/%m/%Y")
```

Comment:
Extract `%x%x%x` to en.yml. Check this [article](http://www.stackednotion.com/blog/2015/01/03/formatting-dates-and-times-in-rails-with-i18n-localize/)